unlock-app/src/App.js

```diff
@@ -1,55 +0,0 @@
-// Modules
```

Comment:
Could you explain how this is being replaced?

Reply:
It's in [`_app.js`](https://github.com/unlock-protocol/unlock/pull/352/files/a8ec7c2b1e07739294535219ad343d71fd2b592b#diff-e0bde51797b5f02e6ca7b1ef80913342) and [`withReduxStore`](https://github.com/unlock-protocol/unlock/pull/352/files/a8ec7c2b1e07739294535219ad343d71fd2b592b#diff-084b584e69b36cca934c42ef18f5faf0)